source/routes/create/month.js

```diff
@@ -0,0 +1,44 @@
+const PouchDB = require('pouchdb');
+PouchDB.plugin(require('pouchdb-find'));
+const db = new PouchDB('db');
+
+/*
+Send in json form of the month page and it gets sent to the database
+
+The month page will be of form: 
+{
+    "user": "dave",
+    "month": "May",
+    "bullets": ["bullet-id1", "bullet-id2", ...]
+}
+*/
+module.exports = {
+    '/create/month': {
+        methods: ['post'],
+        fn: function(req, res, next) {
+            // Check if every field exists, if not, throw error
+            let requiredFields = ["user", "month", "bullets"];
+            requiredFields.forEach((jsonField, index)=>{
```

Comment:
Codacy has a fix for the issue: [Missing space before =>.](https://app.codacy.com/gh/cse110-sp21-group13/cse110-sp21-group13/pullRequest?prid=7423478)
```suggestion
            requiredFields.forEach((jsonField, index) =>{
```